sdk/python/tests/compiler/testdata/pipelineparams.log

```diff
@@ -3,15 +3,9 @@
 [download : main] downloaded-resultout 100% |********************************|    14  0:00:00 ETA
 [download : main] '/tekton/results/downloaded-resultoutput' saved
 
-[download : copy-artifacts] Added `storage` successfully.
-[download : copy-artifacts] tekton/results/downloaded-resultoutput
-[download : copy-artifacts] tar: removing leading '/' from member names
-[download : copy-artifacts] `downloaded_resultOutput.tgz` -> `storage/mlpipeline/artifacts/pipelineparams/download/downloaded_resultOutput.tgz`
-[download : copy-artifacts] Total: 0 B, Transferred: 138 B, Speed: 11.44 KiB/s
-
-[download : sidecar-echo] 2020/07/31 09:29:31 Server is listening on :5678
-[download : sidecar-echo] 2020/07/31 09:29:43 localhost:5678 127.0.0.1:54892 "GET / HTTP/1.1" 200 14 "Wget" 15.199µs
```

Comment:
So, why do these two lines deleted? for my understand, it's not the step of "copy-artifacts". Thanks.

Reply:
They are not deleted. Look at line 7-8 in this file where the date of these logs are updated.